src/app/inbox/components/notification/notification.component.scss-theme.scss

```diff
@@ -11,10 +11,10 @@
   }
   :host.archived .header {
     background-color: map-get($background, background);
-    color: $text-dark-mid !important;
+    color: map-get($foreground, 'text-dark-mid') !important;
```

Comment:
This won't work unless is added to the foreground palette 
https://github.com/ORCID/orcid-angular/blob/bde83b34b66386012920018cc445e0573c83e1d8/src/assets/scss/material.light-theme.scss#L42-L53

Am i wrong?